### DISCUSSION TITLE
Help Watch

### DISCUSSION THREAD

**Discussion by Matti160997** [dawigit/picoclock#10] [Q&A] [answered]
hey hey im a big fan of your work on the watch i was just curious about 3 things how can i rotate the whole screen, how can i stop the earth from moving when i move the watch and how can i remove the thing that shows how the watch is tilted other than that its perfect and how can i change the colors of the our ring/arrows of the watch?

really thank you for this amazing project

**Comment by dawigit** [comment]
Hey Matti!
Which version do you use? Touch or non-touch?
There is no way to change the orientation (rotate by 90°) in any menu in the clock.
The earth/eye backgrounds are meant to be moving and it can't be stopped (in any menu).
The 'gyrocross' can't be hidden in any menu. Do I get you right? You want, instead of the 'gyrocross', just one square that turns 'on' when the watch is not tiltet?
There is no way to change the colors of the clock hands ('pointer','arrow',...) or the 'hour ring' in any menu.

A 'custom theme' should do it - where you can change all the colors with a color selector.
(Touch version)

But if you use the 'pigsh' script from the 'tool' dir, the commands are:

gyrocross off
`gyro 0`

rotate background
`scandir 0` 
`scandir 1` 
`scandir 2` 
`scandir 3` 

You're welcome.

**Reply by Matti160997** [reply]
the only thing i cannot get working when i use pigsh it says /dev/ttyACM0: Permission denied in the terminal how would i fix that?

**Reply by dawigit** [reply]
Are you using a pi? You have to add yourself to the 'dialout' group:
`sudo usermod -a -G dialout matti`
Adds user 'matti' to group 'dialout'.
You should be able to access /dev/ttyACM0 (or whatever) now.
No? Does the device even exist?
`ls /dev/ttyA*` ? (Depending on which Linux you're using device names differ.)
`sudo dmesg` after connecting your device to usb. My last entry is:
`[54317.438818] cdc_acm 1-1.1.2:1.0: ttyACM0: USB ACM device`

Try prepending `sudo` to both, minicom and pigsh:
`sudo minicom -o -D /dev/ttyACM0`
`sudo pigsh 0`
The regular way is adding the user to the 'dialout' group.

**Reply by Matti160997** [reply]
and if i can ask i now have my snapshotcut.bmp what to do now like how can i use it on my watch ?

**Reply by dawigit** [reply, marked as answer]
No you can not. Read 'Changing the background image'!
